src/libraries/System.Utf8String.Experimental/src/System/Runtime/Intrinsics/Intrinsics.Shims.cs

```diff
@@ -148,7 +148,9 @@ internal abstract class AdvSimd : ArmBase
         public new abstract class Arm64 : ArmBase.Arm64
         {
             public static Vector64<byte> AddAcross(Vector64<byte> value) => throw new PlatformNotSupportedException();
+            public static Vector64<byte> MaxAcross(Vector128<byte> value) => throw new PlatformNotSupportedException();
```

Comment:
You don't use `MaxAcross` so might want to delete this.

Reply:
Ah good catch. I was using it in my previous commit before I added the custom `MoveMask` method.

Reply:
We'll eventually replace these temporary files with the corelib ones as soon as @pgovind's [work gets merged](https://github.com/dotnet/runtime/pull/38597/files#diff-e4dd398da5ecc2e03aabbc4b6ae4ccb6R14-R19).